styles.css

```diff
@@ -53,6 +55,7 @@ ul {
   font-weight: normal;
   color: #fff;
   font-size: 20px;
+  width: 566px;
 }
 
```

Comment:
Please align the mailbox icon with other nav items vertically. Kindly see the screenshot below for reference.